IPython/html/services/kernels/handlers.py

```diff
@@ -289,7 +275,5 @@ class StdinHandler(ZMQChannelHandler):
     (r"/api/kernels", MainKernelHandler),
     (r"/api/kernels/%s" % _kernel_id_regex, KernelHandler),
     (r"/api/kernels/%s/%s" % (_kernel_id_regex, _kernel_action_regex), KernelActionHandler),
-    (r"/api/kernels/%s/iopub" % _kernel_id_regex, IOPubHandler),
-    (r"/api/kernels/%s/shell" % _kernel_id_regex, ShellHandler),
-    (r"/api/kernels/%s/stdin" % _kernel_id_regex, StdinHandler)
+    (r"/api/kernels/%s/channels" % _kernel_id_regex, ZMQChannelsHandler),
```

Comment:
Looking back at what I wrote and the code, I am not sure why I wrote it that way.

I was trying to ask if there would then be one websocket per kernel then.